--- udpxy.c.orig	2011-06-17 01:42:22.884801535 +0400
+++ udpxy.c	2011-06-17 01:39:44.744039618 +0400
@@ -269,7 +269,7 @@
 
     assert( (sockfd > 0) && code && reason );
 
-    msglen = snprintf( msg, sizeof(msg) - 1, "HTTP/1.1 %d %s \n\n",
+    msglen = snprintf( msg, sizeof(msg) - 1, "HTTP/1.1 %d %s \nContent-Type:application/octet-stream \n\n",
               code, reason );
     if( msglen <= 0 ) return ERR_INTERNAL;
 
